### PR TITLE
AP-2160 check if value set. If not return false -- contains and not_contains

### PR DESCRIPTION
**Pull request by robertsevernsentient**

### CODE REVIEW

File: src/tests/predicates.test.js [hunk 1]
```diff
@@ -489,6 +489,54 @@ describe('predicates.js', () => {
       assert.equal(1, result.failed.size);
     });
 
+    it('should not filter energidirect user with not_contains when value not set', () => {
```
Comment (mattstrom) [inline src/tests/predicates.test.js:492]: This is a public repo. We should scrub mentions like this.
```suggestion
    it('should not filter energidirect user with not_contains when value not set', () => {
```

File: src/tests/predicates.test.js [hunk 2]
```diff
@@ -489,6 +489,54 @@ describe('predicates.js', () => {
...
+          "query_parameters": {}
+        }
+      };
+      const result = predicates.evaluate(context, predicate);
+      assert(result.rejected);
+      assert.equal(0, result.passed.size);
+      assert.equal(1, result.failed.size);
+    });
+
+    it('should not filter energidirect user with contains when value not set', () => {
```
Comment (mattstrom) [inline src/tests/predicates.test.js:516]:
```suggestion
    it('should not filter user with contains when value not set', () => {
```

**Comment by robertsevernsentient** [comment]
hotfix-v2.2

**Review by mattstrom — APPROVED** [review]

**Pull request merged**